NthKey.xcodeproj/project.pbxproj

```diff
@@ -690,7 +690,7 @@
 				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
-				IPHONEOS_DEPLOYMENT_TARGET = 14.2;
+				IPHONEOS_DEPLOYMENT_TARGET = 13.2;
```

Comment:
What's the reason for lowering the target? Since the app is for early adopters I prefer to start with the newest SDK, to avoid legacy baggage. We can add compatibility for older devices in the future.

Reply:
My device was not updated to 14.2 yet, and 13 seems pretty reasonable major release to work with.

Reply:
Ok, that's fine then. LibWallySwift goes back to 12 I believe, so this shouldn't be an issue.